src/DotNetWorker.Grpc/protobuf/src/proto/FunctionRpc.proto

```diff
@@ -326,7 +326,7 @@ message RpcFunctionMetadata {
   // Properties for function metadata
   // They're usually specific to a worker and largely passed along to the controller API for use
   // outside the host
-   map<string,string> Properties = 16;
+   map<string,string> properties = 16;
```

Comment:
This change is from commit - https://github.com/Azure/azure-functions-language-worker-protobuf/commit/47dc95e8e23b1868dcc10a0af89da4a1551b1c2a